### PR TITLE
#168050014 Profile Information from Travel Request Page

### PR DESCRIPTION
**Pull request by amilykassim**
#### What does this PR do?
- Ability to remember your personal information
#### Description of Task to be completed?
- Give a user the ability to provide passport name and passport number
- Give a user the ability to save his request information so that he won't be required to provide it next time
#### How should this be manually tested?
1. Start by cloning repository
`git clone https://github.com/andela/technites-bn-backend.git`

2. Then checkout to this branch
`git checkout ft-get-profile-information-168050014`

3. Run `npm install`

4. Run `npm run migrate:refresh`

5. Run `npm run seed`.

6. Start server  `npm run  dev`

7. Create a request at this endpoint `api/v1/requests?auto_fill=true` POST method

8. Send a request again without providing the passport name and passport number and the request will be saved with the previous passport information

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168050014](https://www.pivotaltracker.com/story/show/168050014)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/controllers/RequestController.js [hunk 1]
```diff
@@ -122,6 +122,11 @@ class RequestController {
 
     const alreadyExistRequest = await isRequestUnique(request.reason, request.departure_date);
     if (alreadyExistRequest) return res.status(409).json({ status: res.statusCode, error: 'Request already exists based on your reason and departure date' });
+
+    // set the cookies
+    // res.cookie('passport_name', req.body.passport_name);
+    // res.cookie('passport_number', req.body.passport_number);
```
Comment (Jordybastien) [inline src/controllers/RequestController.js]: Please remove commented out codes

**Review by Jordybastien — APPROVED** [review]

**Review by gik-hub — APPROVED** [review]
LGTM

File: src/controllers/RequestController.js (hunk 1 above)
Comment (amilykassim) [reply src/controllers/RequestController.js]: The issue is resolved.

**Review by mucyomiller — APPROVED** [review]
LGTM

**Pull request merged**